### PR TITLE
add valid_device for parity with sprint

### PR DESCRIPTION
**Pull request by georgeismike**
Making attributes match Sprint change in: https://github.com/conduit/conduit-sprint/pull/120

### CODE REVIEW

File: lib/conduit/telispire/parsers/query_device_info.rb [hunk 1]
```diff
@@ -37,6 +37,10 @@ class QueryDeviceInfo::Parser < Parser::Base
       is_valid == "Y"
     end
 
+    attribute :valid_device do
```
Comment (pedromartinez) [inline lib/conduit/telispire/parsers/query_device_info.rb]: This is called device_valid within Sprint parser.

**Comment by pedromartinez** [comment]
Where's sim_valid? Is that a separate API call?

**Comment by georgeismike** [comment]
@pedromartinez yes it's a separate call.

**Comment by pedromartinez** [comment]
@georgeismike Hmm. Then we might need a `collect_sim` method that returns true or false so that Magnetic knows what to do after doing the validate device call. Thoughts?

**Comment by georgeismike** [comment]
If the goal is to keep logic out of magnetic then yes I agree something
like that will help. I am a little confused by having that logic in Conduit
and not Connect but I do agree with the premise.

George Whitaker
VP, Technology
BeQuick Software, Inc.
cell: 561-221-8187

**Comment by pedromartinez** [comment]
The only reason to do it in Conduit is speed. I think that `collect_sim` is a little ambitious, perhaps a more generic `lte_device` and allowing Magnetic to use that as the trigger to ask for the sim will suffice. Whatever the field is, if we add it to both sprint and telispire parsers, we'll be able to get the quick gains of supporting Telispire BYOD without having to come up with an abstraction layer within Connect itself.

**Comment by georgeismike** [comment]
I follow. I agree `lte_device` is better. Very clear cut with TSP.

File: lib/conduit/telispire/version.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module Conduit
   module Telispire
-    VERSION = "1.0.11".freeze
+    VERSION = "1.1.0".freeze
```
Comment (silvahq) [inline lib/conduit/telispire/version.rb]: why 1.1.0 instead of 1.0.12?
what's the logic we use for how to update version?

**Comment by pedromartinez** [comment]
You'll need to add the `lte_device` field if we're going to work this into being supported in Magnetic. Otherwise, it looks good.

File: lib/conduit/telispire/version.rb (hunk 2 above)
Comment (georgeismike) [inline lib/conduit/telispire/version.rb]: new feature being introduced was my logic.

**Comment by pedromartinez** [comment]
Note: you're going to have to add a `sim_valid` on the validate sim action.

**Pull request merged**